src/App.vue

```diff
@@ -286,7 +290,8 @@
 @media only screen and (max-width:1199px) {
   .mapboxgl-ctrl-top-right {
     top: 0;
-    right: calc(100vw - 600px);
+    left: calc(var(--sidebar-width) + #{$spacing-default});
+    right: unset;
```

Comment:
This is unrelated to the update, just a fix to prevent this
![image](https://github.com/openearth/rws-viewer/assets/15196342/aa4d34b5-1570-4c61-b694-f61d9249369b)